app/UI/config-other.ts

```diff
@@ -94,7 +91,6 @@ export class OtherConfig implements PinListGetter, NostrEventAdder {
         if (err instanceof Error) {
             return err;
         }
-        /* no await */ this.nostrEventPusher.put(event);
```

Comment:
Will otherConfig no longer be sent to the pool for synchronization? Maybe we can also remove sync_client_specific_data from app.tsx at the same time.

Reply:
Configuration synchronization is still needed but the old implementation is bad. We need to refactor it